scripts/sanitycheck

```diff
@@ -3208,6 +3210,16 @@ def main():
     global run_individual_tests
     options = parse_arguments()
 
+    if os.name == 'nt':
+        if (not options.build_only) or (not options.disable_size_report):
```

Comment:
nitpicking, but why not have a single `if X and Y` instead of 2 `if` statements?

Reply:
I can update it.  

print("Testing with sanitycheck on Windows is not fully supported yet.")
 ^^ was above the 2nd if before I moved it upstream.